staff/pedrovalverde/css-layout-adaptive/styles/mobile.css

```diff
@@ -0,0 +1,51 @@
+body, html{
+    height: 100%;
+}
```

Comment:
background-color: black; is repeated in all the selectors of your file, like other properties ... should you group properties?

Reply:
yes, I should. thanks 👍